wp-content/themes/core/assets/css/src/theme/typography/_mixins.pcss

```diff
@@ -5,32 +5,202 @@
+}
+
+/* -------------------------------------------------------------------------
+ * Display X-Small
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-display-x-small {
+	font-family: var(--font-family-sans-serif);
+	font-size: 24px;
+	line-height: 1.5;
+	font-weight: 700;
+}
+
+/* -------------------------------------------------------------------------
+ * Display XX-Small
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-display-xx-small {
+	font-family: var(--font-family-sans-serif);
+	font-size: 18px;
+	line-height: 1.56;
+	font-weight: 700;
+	letter-spacing: 0.1111px;
+}
+
+/* -------------------------------------------------------------------------
+ * Body Large
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body-large {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body-large);
+	line-height: 1.9;
+	font-weight: 400;
+}
+
+/* -------------------------------------------------------------------------
+ * Body
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body);
+	line-height: 2;
+	font-weight: 400;
+}
+
+/* -------------------------------------------------------------------------
+ * Body Small
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body-small {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body-small);
+	line-height: 2;
+	font-weight: 400;
+	letter-spacing: 0.1px;
+}
+
+/* -------------------------------------------------------------------------
+ * Overline
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-overline {
+	font-family: var(--font-family-sans-serif);
+	font-size: 14px;
+	line-height: 2;
+	font-weight: 400;
+	letter-spacing: 1.75px;
+	text-transform: uppercase;
+}
+
+/* -------------------------------------------------------------------------
+ * Caption
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-caption {
+	@mixin t-body-small;
+}
+
+/* -------------------------------------------------------------------------
+ * Button
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-button {
+	font-family: var(--font-family-sans-serif);
+	font-size: var(--font-size-body);
+	line-height: 1.125;
+	font-weight: 700;
+	letter-spacing: 0.02em;
+}
+
+/* -------------------------------------------------------------------------
+ * Text Button/Call To Action
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-text-button {
+	font-family: var(--font-family-sans-serif);
+	font-size: 18px;
+	line-height: 1.5556;
+	font-weight: 700;
+	letter-spacing: 0.1111px;
+}
```

Comment:
Probably belong in buttons mixins?

Reply:
I think it makes more sense to keep all the typography mixins with the sink typography. As again, these may be used for other "components" than just buttons.

Reply:
Hmm, don't love the contextual naming of these typographic things (which I see is coming from figma) as it implies...context. I'll stick a note in this to follow up with design on for later, but I see what you're saying and based on how it's setup, agreed.